layouts/register/register-online.html

```diff
@@ -5,7 +5,7 @@
   </header>
   <p class="usa-font-lead">{{ title .Title }} allows residents to register online.</p>
   <p>
-    <a href="{{ .Params.external_link }}">Start your online registration</a>
+    <a href="{{ .Params.external_link }}?ref={{ .Site.Params.Referrer_Short_Code }}">Start your online registration</a>
```

Comment:
@rogeruiz is there a reason we're not using the referrer code in `register-in-person.html` external links?